libgalois/src/DynamicBitset.cpp

```diff
@@ -108,6 +108,21 @@ katana::DynamicBitset::count() const {
   return ret.reduce();
 }
 
+size_t
+katana::DynamicBitset::SerialCount() const {
+  size_t ret = 0;
+  for (uint64_t n : bitvec_) {
+#ifdef __GNUC__
+    ret += __builtin_popcountll(n);
+#else
+    n = n - ((n >> 1) & 0x5555555555555555UL);
```

Comment:
I would prefer to resort to `int _popcnt64 (__int64 a)` intrinsic from `<immintrin.h>`. Perhaps it can be just used always, as it is unlikely that someone will use Katana on a CPU without `POPCNT`. See https://www.intel.com/content/www/us/en/docs/intrinsics-guide/index.html#text=pop&ig_expand=5445